projects/demo/src/modules/components/slider-old/examples/1/index.html

```diff
@@ -0,0 +1 @@
+<tui-slider [formControl]="testValue" [max]="10"></tui-slider>
```

Comment:
unfortunately, git did not properly detect diff :(

All files with old slider's version (their paths includes `*/slider-old/*`) were not touched in this PR.